app/components/validated-input/component.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+const {
+    isEmpty,
+    computed,
+    defineProperty,
+} = Ember;
+
+import Validations from '../../validators/preprint-form-validator';
```

Comment:
Yes, this is a mistake. I originally thought I had to extend the validations in the component in order for the component to work. This will be removed